tests/tests/hrrr_control_decomp

```diff
@@ -0,0 +1,109 @@
+###############################################################################
+#
+#  HRRR control test with different PEs
+#
+###############################################################################
+
+export TEST_DESCR="Compare HRRR decomp results with control"
+
+export CNTL_DIR=hrrr_control
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf009.nc \
+                   sfcf012.nc \
+                   atmf000.nc \
+                   atmf009.nc \
+                   atmf012.nc \
+                   GFSFLX.GrbF00 \
+                   GFSFLX.GrbF09 \
+                   GFSFLX.GrbF12 \
+                   GFSPRS.GrbF00 \
+                   GFSPRS.GrbF09 \
+                   GFSPRS.GrbF12 \
+                   RESTART/coupler.res \
```

Comment:
Same as above, I do not think we need to save restart files in a baseline.

Reply:
This test does not have its own baseline. It uses hrrr_control. If you look at other decomp and 2threads variants, such as cpld_2threads_p8, you will see they do compare the restart files against their control.

```
export CNTL_DIR=hrrr_control
```

Reply:
What I meant to say is that we do not need to have them listed here and compare them against the baselines, if history files are identical. Comparing all these restart files takes time.

Reply:
The rap variants (rap_2threads, etc.) already do those extra comparisons. I want the hrrr tests to match the rap tests.

Reply:
You should open an issue for that.